clusterloader2/pkg/config/cluster.go

```diff
@@ -40,8 +40,12 @@ type ClusterConfig struct {
 	MasterName                 string
 	KubemarkRootKubeConfigPath string
 	DeleteStaleNamespaces      bool
-	IsSSHSupported             bool
-	IsPprofSupported           bool
+	// IsSSHToMasterSupported is false on some managed providers.
+	// Clusterloader will not attempt operations requiring SSH when this value is false, such as some metrics collection routines.
+	IsSSHToMasterSupported bool
+	// IsAPIServerPprofExposed is false for some managed providers.
+	// When IsAPIServerPprofExposed is false, clusterloader will avoid using using pprof to collect metrics.
+	IsAPIServerPprofExposed bool
```

Comment:
Looking at the code, I thin the question is more whether the pprof is enabled in apiserver (we're exposing in on our own in profile.go). Can we change the name to IsAPIServerPprofEnabled and change the doc to say something like
`When IsAPIServerPprofEnabled is false, clusterloader2 will avoid collecting kube-apiserver pprofs`

Reply:
fixed 👍